### PR TITLE
remove inline css/javascripts

### PR DESCRIPTION
**Pull request by gyst**

### CODE REVIEW

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/207/
Test PASSed.

File: src/ploneintranet/messaging/browser/send-message.pt [hunk 1]
```diff
@@ -10,6 +10,8 @@
     metal:fill-slot="main">
 
 
+    <!-- FIXME implementation incomplete/broken -->
+
```
Comment (mattss) [inline src/ploneintranet/messaging/browser/send-message.pt:14]: Can you elaborate on this comment? Do we need a ticket to track this?

Comment (mattss) [inline src/ploneintranet/messaging/browser/send-message.pt:14]: Sorry this question was for @gyst but I forgot to direct it to you.

**Comment by gyst** [comment]
@mattss that FIXME refers to the fact that I'm changing a component that hasn't been properly integrated into ploneintranet yet: ploneintranet.messaging. Just wanted to flag that changing the frontend there should not indicate that that frontend is at all working or supported.

**Pull request merged**